kubeflow/core/pod.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
```

Comment:
What's this for? Aren't use using the K8s job in iap.libsonnet to run update_backend.sh?